zeth_contracts/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "zerocash on Ethereum",
   "main": "truffle.js",
   "scripts": {
+    "install": "ln -s ../../depends/ganache-cli node_modules/ganache-cli && cd node_modules/ganache-cli && npm install",
```

Comment:
Have you thought about using docker to handle our custom `ganache-cli`? Instead of using a submodule, why not add one line to the client README to say:
"""
Fetch ganache with BW6-761/BLS12-377 support:
`docker run --rm -ti clearmatics/experimental-ganache -p 8545:8545`
"""

Reply:
Yes, I did consider that.  It's certainly one option here, with a couple of advantages.
For now I'd like to keep the iteration process as trivial as possible, but once we are happy with the state of this then I think it's a good option to consider.

Reply:
Perfect, I commented on #285 to log this somewhere.